Mage/src/main/java/mage/abilities/effects/common/cost/SpellsCostReductionAllEffect.java

```diff
@@ -137,4 +137,4 @@ public void setControllerId(UUID controllerId) {
     public SpellsCostReductionAllEffect copy() {
         return new SpellsCostReductionAllEffect(this);
     }
-}
+}
```

Comment:
That added line break was automatically added by your IDE I think?
You might want to tweak that (same for the 2 other files)

Reply:
Yeah don't know what the difference was I did modify those files but thought I had reverted them